frontend/src/config/srilanka/layers.json

```diff
@@ -1,7 +1,7 @@
 {
   "admin_boundaries": {
     "type": "boundary",
-    "path": "data/srilanka/lka_bnd_adm3.json",
```

Comment:
should we replace the old file completely?

Reply:
yes, done.